Makefile.am

```diff
@@ -79,14 +80,19 @@ libwd_crypto_la_SOURCES=wd_cipher.c wd_cipher.h wd_cipher_drv.h \
 			wd_digest.c wd_digest.h wd_digest_drv.h \
 			wd_util.c wd_util.h \
 			wd_sched.c wd_sched.h \
-			wd.c wd.h
+			wd.c wd.h \
+			arm_arch_ce.h isa_ce_sm3.h
```

Comment:
Why do you need to build ce header files to libwd_crypto? These files are same with other drviers.

Reply:
done